app/assets/javascripts/request_refund.js

```diff
@@ -0,0 +1,34 @@
+$(document).ready(function () {
+  $('.refund-button').click(function () {
+    var uniq_id = $(this).data('id');
+    $('#uniq_id').val(uniq_id);
+  });
+  $('#reason-label').hide();
+  $('input[type="radio"]').click(function () {
+    if ($('#reason-other').is(':checked')) {
+      $('#reason-label').show();
+    } else {
+      $('#reason-label').hide();
+    }
+  });
+  $('#submit-refund').click(function () {
+    var reason = 'Event was Cancelled';
+    var uniq_id = $('#uniq_id').val();
+    if ($('#reason-custom').val() !== '') {
+      reason = $('#reason-custom').val();
+    }
```

Comment:
You are not checking if an empty reason is provided. So there is no need do the check, just assign `reason = $('#reason-custom').val(); `